dart-devel.rb

```diff
@@ -0,0 +1,52 @@
+class DartBeta < Formula
```

Comment:
Not sure if it matters to homebrew, but this should be DartDev.

Reply:
Oh..good catch